code/game/objects/items/devices/scanners.dm

```diff
@@ -293,23 +293,23 @@ GENE SCANNER
 					max_damage += ", "	//prelude the organ if we've already reported an organ
 					max_damage += organ.name	//this just slaps the organ name into the string of text
 				else
-					max_damage = "\t<span class='alert'>Non-Functional Organs: "	//our initial statement
+					max_damage = "\t<span class='alert'>Non-Functional Organs: </span>"	//our initial statement

```

Comment:
Are you absolutely sure this changes are correct? Shouldn't the </span> be in the next line?

Reply:
You're right, that's a tricky one... I'll change it back just to be safe.